POSTER_PROJECTS/README.md

```diff
@@ -1,52 +1,144 @@
+| 5 | Mutable File System (MFS)| [MFS](topic-MFS.md) |
+| 6 | PeerPad Capabilities System |  |
+| 7 | Pinning & GC on IPFS | [pinning-gc](topic-pinning-gc.md) |
+| 8 | Libp2p Connection Flow | What is the connection flow of a libp2p connection? What happens internally and why? (what is the libp2p-switch) |
+| 9 | IPFS DHT | [IPFS-DHT](topic-IPFS-DHT.md) |
+| 10 | The intricacies of DWeb Addressing | The challenges with the origin policy and all details to have in consideration when adding IPFS natively to Web Browsers resolution. Consider base58 -> base32 migrations, case sensitivity, browser vendors, etc. |
+| 11 | KeyStore & Linked Data Key (aka Multikey) | Self describing keys. What kind of formats are there? How do we handle them all? What do we use them for? |
+| 12 | The design of the IPFS Repo | What is the IPFS Repo? How does it store data? How can users store data in multiple backends (local fs, IndexedDB, S3 and so on) |
+| 13 | DEX - The Importers and Exporters Project | Getting and formatting data into and out of IPFS. |
+| 14 | IPFS/libp2p PubSub | [PubSub](topic-pubsub.md) |
+| 15 | NPM on IPFS | [NPM-on-IPFS](topic-NPM-on-IPFS.md) |
+| 16 | InterPlanetary Test Lab | |
+| 17 | CAR |  Content addressable archive(s). How does it work? What are some applications of it? |
+| 18 | SECIO | Encryption method/protocol. How does it compare to alternatives (like TLS)? |
+| 19 | Rendezvous Protocol | [rendezvous-protocol](topic-rendezvous-protocol.md) |
+| 20 | Circuit Relay | [circuit-relay](topic-circuit-relay.md)|
+| 21 | DNSLink | Utilize DNS to point to IPFS data. |
+| 22 | xTP | External Transport Protocol. How do IPFS nodes communicate with foreign protocols? Why would we need this? |
+| 23 | Go-ds-crdt - Merkle CRDTs |  |
+| 24 | DAG walking facilities in IPLD | [topic-DAG-walking-in-IPLD](topic-DAG-walking-in-IPLD.md) |
+| 25 | Core API | Higher level API for interacting with IPFS nodes. |
+| 26 | New Go HTTP API libs  | Lower level node API. |
+| 27 | QUIC | Transport protocol. When and why is it useful to use QUIC over alternatives? What is the importance of good transports in general? |
+| 28 | CRDTs in Peerpad |  |
+| 29 | Stream Muxers | Libp2p mplex, yamux, SPDY etc. |
+| 30 | Websocket Transport | Why Websockets exist? Why TCP and QUIC can be used everywhere? |
+| 31 | Performance | Where and why can things be slow in a typical IPFS publish or request operation? |
+| 32 | CPU | What are the most compute-heavy operations in a production IPFS node and why? |
+| 33 | IPNS | [IPNS](topic-IPNS.md) |
+| 34 | Routing | How does IPFS handle routing? (how does IPFS abstract peers via IDs instead of location?) What discovery options exist? (kad-dht, rendezvous). How do we take advantage of multiple discovery methods? |
+| 35 | Graphsync | [Graphsync](topic-graphsync.md) |
```

Comment:
@djdv You will need to replace by absolute urls as Github doesn't fancy relative behind the first level readme.

Reply:
That might only be true in this patch view. When merged I think it will work.
See: https://github.com/djdv/camp/tree/feat/poster-session-topics/POSTER_PROJECTS#topics
(the images are also broken in this view but work in the other branch)

I can still change this though if wanted.

Reply:
I've tried this many times before, but maybe something changed

Reply:
Let's merge it and give it a try :)